.changeset/perfect-bags-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/switch': minor
+---
+
+BREAKING: Make keyup handlers protected, enables subclassers to switch using alternative keys.
```

Comment:
It should actually not be considered breaking, right?

Reply:
Ah true, I think this was still from adding the arrow key behaviors, changed